app.rb

```diff
@@ -68,8 +72,51 @@ def new_book
     @books << book
   end
 
+  def list_albums
+    @check.check_list_albums(@albums)
+  end
+
+  def list_genres
+    @check.check_list_genres(@genres)
+  end
+
+  def add_album
+    new_album
+    puts "\nAlbum added successfully."
+  end
+
+  private
```

Comment:
Kindly remove this `private` keyword as it is preventing your @application.finish in your user_interface_actions.rb file from being called. This throws a private method error when the application is exited